### DISCUSSION TITLE
Can you help me?

### DISCUSSION THREAD

**Discussion by matheus0331** [community/community#144988] [npm] [answered]
When I install node_modules using "npm install", I can not do it.
I think package.json file is wrong.
Help me.

**Comment by lucatosc** [comment, marked as answer]
rm -rf node_modules package-lock.json
npm install

You can use this code.

**Comment by AuroraEchos** [comment]
Hey, bro, according to your description, I don't seem to understand what specific problem you have encountered. If you can, you can use (**npm install --verbose**) for more detailed error information. Of course, you can check the package.json file for syntax errors or **missing dependencies**.
